<?xml version="1.0"?>
<launch>
  <arg name="input/main_object" default="main_object"/>
  <arg name="input/sub_object" default="sub_object"/>
  <arg name="output" default="merged_object"/>
  <arg name="data_association_matrix_path" default="$(find-pkg-share m4_tracked_object_merger)/config/data_association_matrix.param.yaml"/>

  <node pkg="m4_tracked_object_merger" exec="m4_tracked_object_merger" name="$(anon m4_tracked_object_merger)" output="screen">
    <remap from="input/main_object" to="$(var input/main_object)"/>
    <remap from="input/sub_object" to="$(var input/sub_object)"/>
    <remap from="output/object" to="$(var output)"/>
    <param from="$(var data_association_matrix_path)"/>
  </node>
</launch>
